src/native/libs/System.Security.Cryptography.Native.Android/pal_jni.c

```diff
@@ -682,7 +682,7 @@ int GetEnumAsInt(JNIEnv *env, jobject enumObj)
     return value;
 }
 
-jint init_library_on_load (JavaVM *vm, void *reserved)
+jint AndroidCryptoNative_InitLibraryOnLoad (JavaVM *vm, void *reserved)
 {
     (void)reserved;
     LOG_INFO("JNI_OnLoad in pal_jni.c");
```

Comment:
nit: this outdated log message might be confusing
```suggestion
    LOG_INFO("AndroidCryptoNative_InitLibraryOnLoad in pal_jni.c");
```

Reply:
I actually planned on removing the message in a separate PR. It serves no real purpose and logcat calls are very expensive.

Reply:
it has sometimes been useful in the past but `LOG_DEBUG` should be enough